tensorflow/core/debug/debug_io_utils.cc

```diff
@@ -147,6 +146,7 @@ Status DebugIO::PublishDebugMetadata(
   Status status;
   for (const string& url : debug_urls) {
     if (str_util::Lowercase(url).find(kGrpcURLScheme) == 0) {
+#if defined(PLATFORM_GOOGLE)
```

Comment:
grpc:// debug URL implementation has two parts: client (debug ops) and server. The server part, which is defined as a Python grpc library, is currently not available in open source. See b/23796275 cited below. So even though the client side has been available in OSS for a while, OSS users haven't been able to use the feature yet to the lack of a working server side. As such, the change here shouldn't affect them.